src/Spice86.Core/CLI/CommandLineParser.cs

```diff
@@ -31,6 +46,21 @@ public class CommandLineParser : ICommandLineParser {
             throw exception;
         });
     }
+    
+    private static string? GetCommandLineName<T>(string propertyName, bool shortName) {
```

Comment:
get -> generate? From what I understand this regenerates the name that could have been passed to the command line for the property?

Reply:
It gets them from the attribute via reflection.

It's an alternative to copying them in this file as '-a' or '-ExeArgs', as it might change.

Reply:
It could use some comments to that effect.

Reply:
Comment added.